components/financial_assistance/config/routes.rb

```diff
@@ -57,4 +57,66 @@
       end
     end
   end
+
+  namespace :api do
+
+    resources :v1 do
```

Comment:
This should probably be `namespace :v1 do` so that we don't create routes for `v1`.